src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -75,6 +79,19 @@ const DetailsWidget = ({
   const Amount = () => values?.amount as ReactElement;
   const Symbol = () => values?.tokenSymbol as ReactElement;
 
+  const showFromDomain = DETAILS_FOR_ACTION[
+    ActionPageDetails.fromDomain
+  ]?.includes(actionType);
+  const showToRecipient = DETAILS_FOR_ACTION[
+    ActionPageDetails.toRecipient
+  ]?.includes(actionType);
+  const showToDomain = DETAILS_FOR_ACTION[ActionPageDetails.toDomain]?.includes(
+    actionType,
+  );
+  const showAmount = DETAILS_FOR_ACTION[ActionPageDetails.amount]?.includes(
+    actionType,
+  );
```

Comment:
In its current form, this is not scalable.

Please refactor this into a helper method similar to those inside `utils/events`, but make it so that it's scalable, so you don't have to change it every time you add new values to the `DETAILS_FOR_ACTION` map.

Something similar to this: https://github.com/JoinColony/colonyDapp/blob/5efc8b7820303b4684b55586102d9aa2931cff24/src/utils/events/index.ts#L100-L107